internal/glrepo/resolver.go

```diff
@@ -37,14 +37,17 @@ func ResolveRemotesToRepos(remotes Remotes, client *gitlab.Client, base string)
 	return result, nil
 }
 
-func resolveNetwork(result *ResolvedRemotes) {
+func resolveNetwork(result *ResolvedRemotes) error {
 	// Loop over at most 5 (maxRemotesForLookup)
 	for i := 0; i < len(result.remotes) && i < maxRemotesForLookup; i++ {
 		networkResult, err := api.GetProject(result.apiClient, result.remotes[i].FullName())
 		if err == nil {
 			result.network = append(result.network, *networkResult)
+		} else {
```

Comment:
Seems like we are taking the fail fast approach and this will fail network resolution if the first remote is not reachable. 

See my comment here: https://github.com/profclems/glab/pull/886#issuecomment-959662401